tests/wsl/enable_systemd.pm

```diff
@@ -17,27 +17,38 @@ sub run {
 
     assert_screen(['windows_desktop', 'powershell-as-admin-window']);
     $self->open_powershell_as_admin if match_has_tag('windows_desktop');
-    # In openSUSE the WSL shell is not root, so there's need to run become_root
-    # in order to write in the /etc/wsl.conf file
+
+    # Check that systemd is not enabled by default.
+    $self->run_in_powershell(
+        cmd => '$port.WriteLine($(wsl /bin/bash -c "systemctl is-system-running"))',
+        code => sub {
+            die("Systemd is running by default...")
+              unless wait_serial("offline");
+        }
+    );
     $self->run_in_powershell(
         cmd => q(wsl),
         code => sub {
-            become_root if (is_opensuse);
-            enter_cmd("ps 1 | grep '/init'");
-            enter_cmd("stat /init | grep 'init'");
-            enter_cmd("echo -e '[boot]\nsystemd=true' > /etc/wsl.conf");
-            wait_still_screen stilltime => 3, timeout => 10;
+            # become_root is now preferred and expected behavior:
+            # https://bugzilla.suse.com/show_bug.cgi?id=1225075
+            become_root;
+            enter_cmd("zypper in -y -t pattern wsl_systemd");
+            wait_still_screen stilltime => 3, timeout => 10, similarity_level => 43;
             save_screenshot;
             enter_cmd("exit");
-            # In openSUSE there's need to exit twice, one from the root and another
+            # There's need to exit twice, one from the root and another
             # one from the WSL
-            enter_cmd("exit") if (is_opensuse);
+            enter_cmd("exit");
```

Comment:
I would repeat something like `wait_still_screen stilltime => 3, timeout => 10, similarity_level => 43;` as we want to type another string after certain synchronization

Reply:
ok, will do!